cheroot/server.py

```diff
@@ -1805,7 +1793,22 @@ def bind(self, family, type, proto=0):
                 # this machine's TCP stack
                 pass
 
+        # So we can reuse the socket...
+        try:
+            if not IS_WINDOWS and family == socket.AF_UNIX:
+                os.unlink(self.bind_addr)
+        except Exception:
+            pass
+
         self.socket.bind(self.bind_addr)
+
+        # So everyone can access the socket...
+        try:
+            if not IS_WINDOWS and family == socket.AF_UNIX:
+                os.chmod(self.bind_addr, 0o777)
```

Comment:
Is 0o777 the right permissions to set unconditionally? Why not let the os set the permissions naturally, and leave it to the client to change the permissions if needed?

Reply:
It guarantees that everyone can access it. Also, this PR is not intended to change the behavior. Only fix a mistake.